bin/index.cjs

```diff
@@ -0,0 +1,147 @@
+#! /usr/bin/env node
+const chalk = require('chalk');
+const PackageJson = require('@npmcli/package-json');
+const readline = require('readline');
+const shell = require('shelljs');
+const { Snippet } = require('enquirer');
+const validatePackageName = require('validate-npm-package-name');
+
+const SUCCESS_CODE = 0;
+const ERROR_CODE = 1;
+
+const SCRIPT_STEPS = [
+  validateEnvironment,
+  cloneRepository,
+  installDependencies,
+  runSetup,
+  cleanup
+];
+
+const rl = readline.createInterface({ input: process.stdin, output: process.stdout });
+const prompt = query => new Promise(resolve => rl.question(query, resolve));
+
+let projectName = '';
+
+const setupSnippet = new Snippet({
+  name: 'package.json',
+  message: 'Fill out the fields in package.json',
+  required: true,
+  template: `{
+  "description": "\${description}",
+  "version": "\${version:0.0.1}",
+  "author": "\${author_name} <\${author_email}> (https://github.com/\${username})"
+}
+`
```

Comment:
@mskorsur Is the indentation here intentional?

Reply:
Yes, it looks weird in the console if it's formatted properly in the code.